src/test/java/pro/sky/calculator/CalculatorServiceTest.java

```diff
@@ -5,22 +5,22 @@
 
 public class CalculatorServiceTest {
 
-    private final CalculatorService calculatorService = new CalculatorService();
+    private final CalculatorServiceImpl calculatorServiceImpl = new CalculatorServiceImpl();
```

Comment:
ссылку лучше делать интерфейсную
```suggestion
    private final CalculatorService calculatorServiceImpl = new CalculatorServiceImpl();
```